docs/flow/reference/components/prometheus.integrations.node_exporter.md

```diff
@@ -0,0 +1,310 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/prometheus.integrations.node_exporter
+title: prometheus.integrations.node_exporter
+---
+
+# prometheus.integrations.node_exporter
+The component embeds the popular
```

Comment:
Part of me worries about using popular here for reference documentation.

Reply:
Also in this first paragraph, we may want to mention what OS(s) it supports, we do later on but having it up front lets people pivot to or from easily.

Reply:
+1 on dropping the popular quantifier